routingpy/routers/openrouteservice.py

```diff
@@ -524,10 +524,6 @@ def matrix(
             Default False.
         :type resolve_locations: bool
 
-        :param units: Specifies the unit system to use when displaying results.
-            One of ["m", "km", "m"]. Default "m".
-        :type units: str
-
```

Comment:
It looks like this is deprecated [docs](https://openrouteservice.org/dev/#/api-docs/v2/matrix/{profile}/post).
<img width="616" alt="Screenshot 2025-06-17 at 20 13 22" src="https://github.com/user-attachments/assets/2f4041e6-dfc3-48ae-8710-e72a492c439e" />

I did not check it against a real server as I don't have access to it. Please let me know if you do.

Reply:
The parameter is used by the server. Will rollback it.